common/api/ecschema-metadata.api.md

```diff
@@ -7,6 +7,7 @@
 import { BaseFormat } from '@itwin/core-quantity';
 import { BentleyError } from '@itwin/core-bentley';
 import { DecimalPrecision } from '@itwin/core-quantity';
+import { FormatProps as FormatProps_2 } from '@itwin/core-quantity';
```

Comment:
I missed this at first but this indicates a problem we should probably solve.  core-quantity and ecschema-metadata both have a definition for FormatProps.  They are the same except that the one in ecschema-metadata extends SchemaItemProps.

We shouldn't have two things with the same name and we should try to avoid duplicating the definitions.  Maybe we could rename ecschema-metadata FormatSchemaItemProps or SchemaItemFormatProps (I don't like either) and it could extend both SchemaItemProps and FormatProps (from core-quantity)

Please check for other duplicate names between the two packages.

Reply:
Done!

Reply:
`type SchemaItemFormatProps = FormatProps & SchemaItemProps` would be clearer and less error-prone than duplicating the big list of properties.

Reply:
Done

Reply:
Thanks. You good @ColinKerr?